samples/controller.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cosi-controller-manager
+  labels:
+    run: COSIController
```

Comment:
should this be `app: cosi-controller`? 

I just haven't seen this convention before.